src/Storageless/Http/SessionMiddleware.php

```diff
@@ -248,25 +248,21 @@ public function extractSessionContainer(Token $token = null) : SessionInterface
     private function appendToken(SessionInterface $sessionContainer, Response $response, Token $token = null) : Response
     {
         $sessionContainerChanged = $sessionContainer->hasChanged();
-        $sessionContainerEmpty   = $sessionContainer->isEmpty();
```

Comment:
Seen now - this is described in https://github.com/psr7-sessions/storageless/issues/63

 > do we want to keep this lazy initialisation thing or not?

Yes, reading/writing to session in each request, as well as running crypto, is otherwise a very good way to keep data-centers warm. Let's not do that :-)